core/src/epicli/cli/helpers/objdict_helpers.py

```diff
@@ -32,6 +33,28 @@ def merge_objdict(to_merge, extend_by):
         if key in to_merge:
             if isinstance(to_merge[key], ObjDict):
                 merge_objdict(to_merge[key], val)
+            elif isinstance(to_merge[key], list):
```

Comment:
This function is quite long, you can consider splitting to smaller ones to have self-documenting code (they would have names) or add a comment under added part for people like me :)

Reply:
- Split up function
- Documented it
- Added and restructured tests to reflect the new changes

Reply:
Broke it down, documented it a bit, added more tests to cover some edge case:)